doc/reference/std/misc/list-builder.md

```diff
@@ -0,0 +1,68 @@
+# List builder
+
+## with-list-builder ##
+
+``` scheme
+(with-list-builder (put! [peek]) body ...) -> list
+
+  put! := function identifier that modifies internal list
+  peek := optional function identifier that retrieves internal list
+```
+
+A more efficient Syntax sugar for the `call-with-list-builder` procedure,
+so *put!* and *peek* can be used without wrapping them in a lambda first.
+`with-list-builder` returns the internal list at the end.
+
+::: tip Examples:
+``` scheme
+> (import :std/iter)
+> (with-list-builder (put!)
+    (for (n (in-iota 100 1))
+      (let ((mod3 (zero? (modulo n 3)))
+            (mod5 (zero? (modulo n 5))))
+        (put! (cond ((and mod3 mod5) "fizzbuzz")
+                    (mod3 "fizz")
+                    (mod5 "buzz")
+                    (else n))))))
+(1 2 "fizz" 4 "buzz" "fizz" ... 97 98 "fizz" "buzz")
+```
+:::
+
+## call-with-list-builder ##
+
+## call-with-list-builder ##
```

Comment:
Whats with the ## in the end? Is this even valid markdown?

Can you check that it renders correctly in the netlify preview?